docs/src/pages/demos/progress/progress.md

```diff
@@ -45,3 +45,11 @@ For example, a refresh operation should display either a refresh bar or an activ
 ### Query
 
 {{"demo": "pages/demos/progress/LinearQuery.js"}}
+
+## Delaying appearance
+
+There is a [school of thought](http://www.nngroup.com/articles/response-times-3-important-limits/) that feedback after user operations isn't necessary for about a second.
+To implement this, you can take advantage of [theme overrides](/customization/themes/#customizing-all-instances-of-a-component-type)
```

Comment:
I don't see how the DelayingAppearance example above relates to theme overriding (I guess theme overriding was more for https://github.com/mui-org/material-ui/issues/9306#issuecomment-350120506)

Reply:
@caub Fixed here: #10147, needs to be fixed again here: #10151

#codefight 😄